src/Refactoring-Core/RBRefactoring.class.st

```diff
@@ -132,6 +129,18 @@ RBRefactoring >> copyOptionsFrom: aDictionary [
 	self options: dict
 ]
 
+{ #category : 'preconditions' }
+RBRefactoring >> failedApplicabilityPreconditions [
+
+	[self checkApplicabilityPreconditions] 
+		on: RBRefactoringError
+		do: [ :err | 
+			^ err failedConditions
+			].
+		^ #()
```

Comment:
I suggest changing this to:

```suggestion
	^ self applicabilityPreconditions reject: [ :cond | cond check ]
```

This way we avoid error handling, which is something we want to achieve and avoid all errors in the refactorings.

Reply:
done :)